### PR TITLE
Speculative password import prompt crash fix

### PR DESCRIPTION
**Pull request by graeme**
Task/Issue URL: https://app.asana.com/0/1203822806345703/1208648341254407/f

**Description**:

Just a BSK bump for a macOS crash fix.

**Steps to test this PR**:
Just green CI

**Definition of Done (Internal Only)**:

* [ ] Does this PR satisfy our [Definition of Done](https://app.asana.com/0/1202500774821704/1207634633537039/f)?

**Copy Testing**:

* [ ] Use of correct apostrophes in new copy, ie `’` rather than `'`

**Orientation Testing**:

* [ ] Portrait
* [ ] Landscape

**Device Testing**:

* [ ] iPhone SE (1st Gen)
* [ ] iPhone 8
* [ ] iPhone X
* [ ] iPhone 14 Pro
* [ ] iPad

**OS Testing**:

* [ ] iOS 15
* [ ] iOS 16
* [ ] iOS 17

**Theme Testing**:

* [ ] Light theme
* [ ] Dark theme

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)

### CODE REVIEW

File: DuckDuckGo.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -10991,8 +10991,8 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/DuckDuckGo/BrowserServicesKit";
 			requirement = {
-				kind = exactVersion;
-				version = 209.0.0;
+				branch = "graeme/speculative-password-import-prompt-crash-fix";
```
Comment (ankushkushwaha) [inline DuckDuckGo.xcodeproj/project.pbxproj]: Should not we wait to merge `graeme/speculative-password-import-prompt-crash-fix` into main branch and then refer the version, as previously done?

Comment (graeme) [reply DuckDuckGo.xcodeproj/project.pbxproj]: @ankushkushwaha The `BrowserServicesKit` changes and the changes in this PR are dependent. It doesn't really make sense to create a new `BrowserServicesKit` release until it's all reviewed so, for now, pointing to a branch makes sense. Once the PR is approved, then of course we will point to a released `BrowserServicesKit` version.

**Review by amddg44 — APPROVED** [review]
The unit tests timed out on the first pass (unrelated to this), all good on the second run

**Comment by graeme** [comment]
This PR was superceded already and BSK has been updated. So I'll just close

**Pull request closed without merging**